jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/AttachHandler.java

```diff
@@ -31,8 +31,12 @@
 import java.io.File;
 import java.io.IOException;
 import java.io.PrintStream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.attribute.FileTime;
 import java.util.Properties;
 import java.util.Vector;
+import java.util.concurrent.TimeUnit;
```

Comment:
While you're here, could you please fix the initializer for `shutdownTimeoutMs`; it should use `Long.getLong()` instead of `Integer.getInteger()`.

Reply:
Fixed